plaintext/2023/05/kasi-anpa-pona.md

```diff
@@ -9,6 +9,7 @@ date-precision: day
 tags:
   - translation
   - music
+  - vocal sythesizer
```

Comment:
sy***n***thesizer ya silly goose
also this isn't really a trait of the text itself so idk if it's desirable to have as a tag? i'm very soft on that point though